LayoutTests/TestExpectations

```diff
@@ -6206,3 +6206,9 @@ webkit.org/b/245905 imported/w3c/web-platform-tests/css/css-text/line-breaking/l
 
 # This test can cause OOM error, depending on device and memory usage, and there is no guarantee. Ignore console log, which appears when OOM happens since what we would like to ensure is just no crash.
 js/dom/Promise-reject-large-string.html [ DumpJSConsoleLogInStdErr ]
+
+# WPT CSS2 tests are not valid anymore
```

Comment:
Do they need upstream?

Reply:
https://wpt.fyi/results/css/CSS2/syntax?label=experimental&label=master&aligned&view=subtest

Reply:
Can you file an issue against the WPT repo?

Reply:
Eventually yes, when I'm done with this import